docs/tutorial-create.rst

```diff
@@ -74,50 +74,86 @@ applied on the server.
   modify ``created`` or ``contributor`` fields. You will get an error if you
   try.
 
-Annotate Samples and Data
-=========================
-
-The obvious next thing to do after uploading some data is to annotate it.
-Annotations are encoded as bundles of descriptors, where each descriptor
-references a value in a descriptor schema (*i.e.* a template). Annotations for
-data objects, samples, and collections each follow a different descriptor
-format. For example, a reads data object can be annotated with the 'reads'
-descriptor schema, while a sample can be annotated by the 'sample' annotation
-schema. Each data object that is associated with the sample is also connected
-to the sample's annotation, so that the annotation for a sample (or collection)
-represents all Data objects attached to it. `Descriptor schemas`_ are described
-in detail (with `accompanying examples`_) in the
-`Resolwe Bioinformatics documentation`_.
+Annotate Samples
+================
 
-.. _Resolwe Bioinformatics documentation: http://resolwe-bio.readthedocs.io
-.. _Descriptor schemas: https://resolwe-bio.readthedocs.io/en/latest/descriptor.html
-.. _accompanying examples: https://github.com/genialis/resolwe-bio/tree/master/resolwe_bio/descriptors
+The next thing to do after uploading some data is to annotate samples this data
+belongs to. This can be done by assigning a value to a predefined field on a
+given sample. See example below.
 
-Here, we show how to annotate the ``reads`` data object by defining the
-descriptor information that populates the annotation fields as defined in the
-'reads' descriptor schema:
+Each sample should be assigned a species. This is done by attaching the
+``general.species`` field on a sample and assigning it a value, e.g.
+``Homo sapiens``.
 
 .. literalinclude:: files/tutorial-create.py
-   :lines: 33-42
+   :lines: 33
+
+
+Annotation Fields
+-----------------
 
-We can annotate the sample object using a similar process with a 'sample'
-descriptor schema:
+You might be wondering why the example above requires ``general.species`` string
+instead of e.g. just ``species``. The answer to this are ``AnnotationFields``\ s.
+These are predefined set of names that are available to annotate samples. Let's
```

Comment:
Annotation field represents more than just a name. Maybe predefined set of attribute/characteristics/terms?

Reply:
Valid point, thanks. It tried to generalize the description without making it too complex.